features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelineView.kt

```diff
@@ -161,28 +161,28 @@ fun TimelineItemRow(
             )
         }
         is TimelineItem.Event -> {
-            fun onClick() {
+            val onClicked = {
                 onClick(timelineItem)
             }
 
-            fun onLongClick() {
+            val onLongClicked = {
```

Comment:
This is also done in #771 , we may have conflict here...

Reply:
[2ad307d](https://github.com/vector-im/element-x-android/pull/778/commits/2ad307dfc2dccf3974e319832b2440a99f93b3d7)